llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -44,7 +44,7 @@ AllowStridedPointerIVs("lv-strided-pointer-ivs", cl::init(false), cl::Hidden,
                                 "pointer induction variables."));
 
 namespace llvm {
-cl::opt<bool>
+static cl::opt<bool>
```

Comment:
Should pull these out of the namespace to be with the other opts. I assume the intent was these should be exposed at some point?

Reply:
I'm not sure, does @jurahul have any thoughts?

Reply:
Agreed, we should pull them out of the namespace

Reply:
Every option inside all namespaces?

Reply:
In general yes

Reply:
I've checked the 52 changed files and take those inside llvm namespace out

Reply:
Since some options are prefixed with namespace when used, this causes issues if I move them out. I'll just leave those as they are.